src/utils/ethereum.ts

```diff
@@ -0,0 +1,3 @@
+import { isAddress, toBN, isBN, isBigNumber, fromAscii, toHex, toWei, fromWei } from 'web3-utils'
+
+export { isAddress, toBN, isBN, isBigNumber, fromAscii, toHex, toWei, fromWei }
```

Comment:
Good one! I think it was the right move to move these utils from the API to here.